plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-terms-block/edit.tsx

```diff
@@ -126,9 +127,26 @@ export const Edit = ( {
 							} )
 						}
 					/>
+					<ToggleControl
+						label={ __(
+							'Show separator above block',
```

Comment:
What are your thoughts on changing this to `Show separator`? I have two reasons for this suggestion:
1. In some languages, translations might become very lengthy, e.g. in German `Show separator above block` would become `Trennzeichen über dem Block anzeigen`, whereas `Show separator` would be `Trennzeichen anzeigen`.
2. For `Require checkbox` we also don't specify where the checkbox will become visible. As soon as a merchant toggles the checkbox, it will immediately become clear where the change happens.